src/community/gwc-azure-blob/src/main/resources/GeoServerApplication.properties

```diff
@@ -0,0 +1,24 @@
+#Generated by Eclipse Messages Editor (Eclipse Babel)
+
+AzureBlobStorePanel.container = Container *
+AzureBlobStorePanel.container.title = The name of the Azure container where the tiles are stored.
+AzureBlobStorePanel.accountName = Account name*
+AzureBlobStorePanel.accountName.title = The Azure Storage Account Name.
+AzureBlobStorePanel.accountKey = Account Key *
+AzureBlobStorePanel.accountKey.title = The Azure Storage Account Key.
+AzureBlobStorePanel.prefix = Azure Object Key Prefix
+AzureBlobStorePanel.prefix.title = A prefix path to use as the root to store tiles under the bucket.
```

Comment:
Update to match whatever you decided to call the storage / container.

Reply:
Container